### PR TITLE
test(push): add ut for source:push:add

### PR DESCRIPTION
**Pull request by olamothe**
## Proposed changes

Adding UT for `:add` command. 

## Testing

- [X] Unit Tests:

https://coveord.atlassian.net/browse/CDX-473

### CODE REVIEW

File: packages/cli/src/commands/source/push/add.spec.ts [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+        errorCode,
+        message,
+      },
+    },
+  });
+
+  const doMockAxiosSuccess = (status: number, text: string) => ({
+    status,
+    statusText: text,
+  });
```
Comment (louis-bompart) [inline packages/cli/src/commands/source/push/add.spec.ts:34]: Let's refactor that I think ;)

It's already in `delete.spec.ts`

File: packages/cli/src/commands/source/push/add.ts [hunk 2]
```diff
@@ -62,9 +62,10 @@ export default class SourcePushAdd extends Command {
     const {flags} = this.parse(SourcePushAdd);
     if (!flags.file && !flags.folder) {
       this.error(
-        'You must provide either --file= or --folder=. See source:push:add --help for more information.'
+        'You must minimally set the `file` or the `folder` flag. Use `source:push:add --help` to get more information.'
```
Comment (louis-bompart) [inline packages/cli/src/commands/source/push/add.ts:65]: Maybe me, but doesn't sounds... "right". Invoking @jpmarceau :P

**Review by louis-bompart — COMMENTED** [review]
lgtm overall.

Needs a refactor, plus san-check on the string

File: packages/cli/src/commands/source/push/add.ts (hunk 2 above)
Comment (olamothe) [reply packages/cli/src/commands/source/push/add.ts:65]: I copied the message from `delete`, which had been revised by Fred Beaudoin.

File: packages/cli/src/commands/source/push/add.spec.ts (hunk 1 above)
Comment (olamothe) [reply packages/cli/src/commands/source/push/add.spec.ts:34]: Indeed !

File: packages/cli/src/commands/source/push/add.ts (hunk 2 above)
Comment (louis-bompart) [reply packages/cli/src/commands/source/push/add.ts:65]: minimally sounds odd, maybe it was just me okok, can be ignored 👍

**Review by louis-bompart — APPROVED** [review]

**Pull request merged**